reactor-netty-http/src/main/java/reactor/netty/http/server/Http2StreamBridgeServerHandler.java

```diff
@@ -40,17 +42,20 @@
 	final ServerCookieEncoder                                     cookieEncoder;
 	final ConnectionObserver                                      listener;
 	final BiFunction<ConnectionInfo, HttpRequest, ConnectionInfo> forwardedHeaderHandler;
+	final Function<? super Mono<Void>, ? extends Mono<Void>> mapHandle;
```

Comment:
nitpick: breaking the nice alignment here :p

Reply:
fixed